chemie/rentalservice/views.py

```diff
@@ -71,6 +71,7 @@ def edit_rentalobject(request, rentalobject_id):
     return render(request, "rentalservice/new_object.html", context)
```

Comment:
This doesn't work since the `RentalObjectForm` used here is not the same as `NewRentalObjectForm` which is used in the `new_object.html` template. The forms should be equal, so that the variables accessed in the template actually exist.